### PR TITLE
Added avatar

### PR DESCRIPTION
**Pull request by fahd23**
Added avatar components and refactored index.html

### CODE REVIEW

**Comment by ozairkhan2810** [comment]
It is suggested to deploy, so that it will be easy to review.
Folder structure looks good.

File: Components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.avatar {
+  width: 45px;
+  height: 45px;
+  border-radius: 50%;
+}
+.sm {
+  width: 30px;
+  height: 30px;
+}
+.lg {
```
Comment (JantuDeb) [inline Components/avatar.css:10]: Be more specific with class names, here it is unclear whether sm, lg, m is for padding, margin, or width. `avatar-sm` would be more explanatory.

**Review by JantuDeb — APPROVED** [review]
I have added a comment look into it, you want to include those changes if you want to.

File: Components/avatar.css (hunk 1 above)
Comment (fahd23) [reply Components/avatar.css:10]: Thanks Jantu, will correct it

**Pull request merged**